ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/LibIpaMultipointTest.java

```diff
@@ -17,36 +17,14 @@
 
 import org.apache.tuweni.bytes.Bytes;
 import org.hyperledger.besu.nativelib.ipamultipoint.LibIpaMultipoint;
-import org.junit.BeforeClass;
 import org.junit.Test;
 
-import java.io.File;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
 public class LibIpaMultipointTest {
 
-  @BeforeClass
-  public static void setUp() {
-    Path buildPath = Paths.get(LibIpaMultipointTest.class.getProtectionDomain().getCodeSource().getLocation().getFile()).getParent().getParent().getParent();
-    File macFile = new File(buildPath.resolve("darwin/lib").toFile(), "libipa_multipoint_jni.dylib");
-    File linuxFile = new File(buildPath.resolve("linux-gnu-x86_64/lib").toFile(), "libipa_multipoint_jni.so");
-    File linuxArmFile = new File(buildPath.resolve("linux-gnu-aarch64/lib").toFile(), "libipa_multipoint_jni.so");
-    if (linuxFile.exists()) {
-      System.load(linuxFile.getAbsolutePath());
-    } else if (linuxArmFile.exists()) {
-        System.load(linuxArmFile.getAbsolutePath());
-    } else if (macFile.exists()) {
-      System.load(macFile.getAbsolutePath());
-    } else {
-      throw new RuntimeException("could not setup jni path for test");
-    }
-  }
-
   @Test
-  public void testCallLibrary() {
+  public void testCallLibrary() throws Exception {
```

Comment:
?? Why add the generic `throws Exception` ?? None of the called apis were changed.

Reply:
Probably an artifact from the back and forth during testing. I can remove that change, sure.